src/workflows-app/components/FilterableWorkflowTable.ts

```diff
@@ -35,6 +45,13 @@ type FilterableWorkflowTableProps = {
   namespace: string;
   submissionId: string;
   workspaceName: string;
+  workspaceId: string;
+  setShowLog: (showLog: boolean) => boolean;
+  setLogsModalTitle: (modalTitle: string) => boolean;
+  setLogsArray: (logsArray: [{}]) => boolean;
+  setTaskDataTitle: (dataTitle: string) => boolean;
+  setTaskDataJson: (data: {}) => boolean;
+  setShowTaskData: (showData: boolean) => boolean;
```

Comment:
`setState` functions do not have a return value.
https://react.dev/reference/react/useState#setstate-returns

```suggestion
  setShowLog: (showLog: boolean) => void;
  setLogsModalTitle: (modalTitle: string) => void;
  setLogsArray: (logsArray: [{}]) => void;
  setTaskDataTitle: (dataTitle: string) => void;
  setTaskDataJson: (data: {}) => void;
  setShowTaskData: (showData: boolean) => void;
```